src/CodeFormatter.cs

```diff
@@ -114,6 +114,9 @@ internal static class CodeFormatter
                 // This flag is used at restore time to avoid imports from packages changing the inputs to restore,
                 // without this it is possible to get different results between the first and second restore.
                 { "ExcludeRestorePackageImports", bool.TrueString },
+                // Clearing VSToolsPath will cause ASP.NET full framework projects to skip loading Microsoft.WebApplication.targets
+                // which does not exist in the .NET Core SDK.
+                { "VSToolsPath", string.Empty }
```

Comment:
@jmarolf Not sure what else might fallout because of this change

Reply:
I would argue the correct fix is to do what Omnisharp does and deploy its own MSBuild that can load legacy tasks and targets